bin/pycbc_banksim

```diff
@@ -222,6 +223,14 @@ parser.add_argument("--mchirp-window", type=str, metavar="FRACTION",
                          "comma separated numbers to have different bounds "
                          "above and below the signal's, with below bound "
                          "listed first.")
+parser.add_argument("--tau0-window", type=float, metavar="TIME", default=None,
```

Comment:
It's in seconds, so I'll add that. But I don't really think I can put any sort of sane value in the help. It depends very much on noise curve and the user *should* check the values they are using, not just trust my input here. (Same is true for mchirp-window, 0.5 is massive overkill for BNS but not enough for IMBH).

.... Anyway the cynical side of me says that the more effort we put into making this code easier to use, the more likely someone will just pick it up and use it without any credit going to the developers.